### PR TITLE
Enhance filtering of AbuseReport entries in /admin

### PR DESCRIPTION
**Pull request by johngian**
* Link to UserProfile on site
* Show profile date joined/last updated
* Filter based on skills

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/users/admin.py [hunk 1]
```diff
@@ -721,10 +721,38 @@ class Meta:
...
 class AbuseReportAdmin(admin.ModelAdmin):
     form = AbuseReportAutocompleteForm
-    list_display = ['profile', 'reporter', 'type', 'created', 'updated']
-    list_filter = ['type', 'is_akismet']
+    list_display = ['profile', 'reporter', 'type', 'created', 'updated',
+                    'profile_last_updated', 'profile_date_joined']
+    list_filter = ['type', 'is_akismet', SkillsFilter]
+    view_on_site = True
+
+    def view_on_site(self, obj):
```
Comment (akatsoulas) [inline mozillians/users/admin.py]: Is this for redirecting to the profile in the site? If yes, where should be the link displayed?

Comment (johngian) [inline mozillians/users/admin.py]: it is displayed, there is a link "view on site"

Comment (akatsoulas) [inline mozillians/users/admin.py]: Ah, missed that, thanks!

**Pull request merged**